BedrockServer.cpp

```diff
@@ -194,7 +194,12 @@ void BedrockServer_WorkerThread(void* _data) {
             data->server->pollTimer.start();
             S_poll(fdm, max(nextActivity, now) - now);
             data->server->pollTimer.stop();
-            nextActivity = STimeNow() + STIME_US_PER_S; // 1s max period
+            nextActivity = STimeNow();
+            // Only wait up to one second if there's nothing in the node's internal queue. Otherwise, we'll want to
+            // process that immediately if no new work is waiting.
+            if (node.isQueuedCommandMapEmpty()) {
+                nextActivity += STIME_US_PER_S; // 1s max period
+            }
```

Comment:
This is a nitpick, but I find this code a little confusing to follow.  Would you be open to:

```
...
data->server->pollTimer.stop();

// nextActivity contains the timestamp of the next action we intend to do.  We use this
// to determine how long to wait for new work to come in from the network.  If our
// command map is empty, then we have nothing in particular to do right now, and can
// wait a second before we check some internal timers to see if we spontaneously 
// decide to do something. On the other hand, if the command map is *not* empty,
// it means we have work to do right now, and thus shouldn't wait at all for the
// network (though we should still check it to see if higher priority work has come in
// while processing the last command).
nextActivity = STimeNow();
if (node.isQueuedCommandMapEmpty()) {
    nextActivity += STIME_US_PER_S; // 1s max period
}

// Handle any HTTPS requests from our plugins.
...
````